packages/@aws-cdk/aws-kinesis/lib/stream.ts

```diff
@@ -152,10 +163,13 @@ abstract class StreamBase extends Resource implements IStream {
     const ret = this.grant(
       grantee,
       'kinesis:DescribeStream',
```

Comment:
I'd love to extract this array to `READ_OPERATIONS` and `WRITE_OPERATIONS` and use those constants in `grantRead` and `grantWrite` accordingly. 

Actually ideally the implementation of `grantReadWrite` would just be:

```typescript
this.grantRead()
this.grantWrite()
```

Is that possible? Not sure what the return value will be in such a case.

Reply:
took a shot at it, let me know what you think